08LessonEight/in-class-project/homework.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Blog</title>
+    <link rel="stylesheet" href="css/homework-style.css">
+</head>
+
+<body>
+
+
+    <header>
+        <h1>Recent From Blog</h1>
+        <div class="centered">
+            <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam fringilla dolor diam, at tempor lectus viverra id. Vestibulum ante ipsum primis in faucibus.
+            </p>
+        </div>
+    </header>
+
+    <main>
+        <!-- This is the left section with the keyboard picture -->
+        <section class="section1">
+            <div>
+                <img src="img/keyboard.jpg" class="large-image" />
```

Comment:
this image has a .JPG extension and needs to be capitalized here as well for it to show on all browsers. other than this, great work as usual.